### ISSUE TITLE
feature: Support  GlslangToSpv

### ISSUE THREAD

**Issue by longde123** [haxiomic/haxe-glsl-parser#15]
have a plan? GLSL ES  -> SPIRV   transpilation   :) 

like this 

https://github.com/KhronosGroup/glslang/blob/master/SPIRV/GlslangToSpv.cpp

**Comment by haxiomic** [comment]
That would be _sweet_, It'd take a while though. Easiest thing to do in the short term is to use a macro to execute linked the code GLSL -> SPRIV as an external process. Another thing worth looking into is [HXSL](https://github.com/HeapsIO/heaps/tree/master/hxsl), which is a haxe-based shader language designed for shader or bytecode generation (see the AgalOut.hx for flash shader bytecode)

**Comment by longde123** [comment]
I think  a backend  SPIRV generate more fun

SPIR-V based GLSL cross-compiler
like this
https://github.com/Kode/krafix

**Comment by longde123** [comment]
whole SPIRV   work process in haxe

**Comment by longde123** [comment]
Good idea for flash bytecode   ,    backend   HXSL  only  need GLSL ES  ast   -> HXSL ast
